playground/config/__tests__/load.spec.ts

```diff
@@ -1,7 +1,11 @@
-import { resolve } from 'node:path'
+import { dirname, resolve } from 'node:path'
+import { fileURLToPath } from 'node:url'
 import { loadConfigFromFile } from 'vite'
 import { expect, it } from 'vitest'
 
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = dirname(__filename)
```

Comment:
Similarly here too.

Reply:
Yeah, could remove this for neat code. However, I think it's not for vite config testing but just using the shim from vitest, and for others spec files. 🤔

Reply:
I think I got this mixed up too.